website/docs/components/modal/partials/guidelines/guidelines.md

```diff
@@ -226,107 +274,38 @@ This action may impact areas of the application outside of the scope of the curr
-    - While certain experiences may only call for a single action button, most Modal instances should have a “cancel” button as a more explicit method to close the Modal.
-
-The entry point for a Modal should be straightforward enough that if a user accidentally dismisses it, the Modal can be easily triggered again.
-
-### Dismissal actions
-
-The following actions will dismiss or close the Modal except if `isDismissDisabled` is set to true in the production component:
+The following actions will dismiss or close the Modal unless `isDismissDisabled` is set to true:
 
 - Dismiss button in the Modal header
-- Clicking with a mouse outside of the Modal on the overlay
+- Clicking outside the Modal on the overlay
 - Hitting the escape (`ESC`) key on the keyboard
-- A cancel or close button in the footer can also act as a way to dismiss the Modal, but doesn’t always function in the same way (see the section on actions for more details).
-
-![Dismissal options for the Modal](/assets/components/modal/modal-dismissal-actions.png)
-
-If a user attempts to dismiss a Modal that contains a partially filled form or other interactive elements, ensure that the following criteria are met to promote a positive user experience:
-
-- The default browser notification is being triggered calling attention to a potential misstep.
-- Partially filled form data is persisted within the application to prevent duplicative work.
-
-!!! Info
+- A cancel or close button in the footer can also act as a way to dismiss the Modal, but doesn’t always function in the same way.
 
-Note: Both of the above steps are recommendations of the HDS team, but are the responsibility of consumers to implement at the application level.
-
-!!!
+![Dismissal options for the Modal](/assets/components/modal/modal-dismissal-actions.png =798x*)
 
-![Default browser notification](/assets/components/modal/modal-default-browser-notification.png)
+In addition to having proper ways for the user to dismiss a Modal, ensure the entry point for a Modal should also be straightforward enough that if a user accidentally dismisses it, the Modal can be easily triggered again.
 
-## Position and responsive sizing
+### Via dismiss button
 
-![Modal positioning](/assets/components/modal/modal-positioning.png)
+The most common dismissal method for a Modal is via the dismiss button in the Modal header. This acts as an escape hatch for the user and helps prevent the user from getting stuck. 
 
-Modals should be positioned in the center of the viewport, paired with the overlay component, and on top of the main page content.
+While we typically don't recommend removing or hiding the dismiss button, it can be done to elevate the importance of the Modal and lower the possibility of accidental dismissal, in which case, a “cancel” button could be added as an escape hatch.
```

Comment:
```suggestion
While we typically don’t recommend removing or hiding the dismiss button, it can be done to elevate the importance of the Modal and lower the possibility of accidental dismissal, in which case, a “cancel” button could be added as an escape hatch.
```